jquery.bootstrap-growl.coffee

```diff
@@ -8,7 +8,10 @@ $.bootstrapGrowl = (message, options) ->
   $alert = $("<div>")
   $alert.attr "class", "bootstrap-growl alert"
   $alert.addClass "alert-" + options.type if options.type
-  $alert.append "<a class=\"close\" data-dismiss=\"alert\" href=\"#\">&times;</a>" if options.allow_dismiss
+  if options.icon
+    $alert.append "<i class=\""+options.icon+"\"></i> "
+  if options.allow_dismiss
+    $alert.append "<a class=\"close\" data-dismiss=\"alert\" href=\"#\">&times;</a>" if options.allow_dismiss
```

Comment:
the "if options.allow_dismiss is duplicated as the if statement above tests for the same value...